DenunciaCiudadana/src/Login.js

```diff
@@ -0,0 +1,50 @@
+import React, {useEffect} from 'react';
+import {View, Text, Button} from 'react-native';
+import auth from '@react-native-firebase/auth';
+import firestore from '@react-native-firebase/firestore';
+
+const Login = ({user}) => {
+  useEffect(() => {
+    async function getComplaints() {
+      const complaints = await firestore().collection('complaints').get();
+      complaints.forEach(documentSnapshot => {
+        console.log(documentSnapshot.data());
+      });
+    }
+    getComplaints();
+  }, []);
+
+  const handleSalir = () => {
+    auth()
+      .signOut()
+      .then(() => console.log('User signed out!'));
+  };
+
+  const crearNota = user => {
+    firestore()
+      .collection('complaints')
+      .add({
+        title: 'Funcionario público es visto comprando votos',
+        description: 'Un funcionario público fue visto comprando votos',
+        area: 'Goverment',
+        like: 0,
+        signedBy: true,
+        user: 'users/' + user.uid,
+        dateEvent: Date.now(),
+        timestamp: Date.now(),
+      })
```

Comment:
Why these values are hardcoded? 🤔

Reply:
It is just to make sure thath the db is working, but in the future we are going to use redux for that oparation